pygmt/datasets/samples.py

```diff
@@ -324,37 +324,15 @@ def load_hotspots(**kwargs):
     return data
 
 
-def load_mars_shape(**kwargs):
+def _load_mars_shape():
     """
-    (Deprecated) Load a table of data for the shape of Mars.
-
-    .. warning:: Deprecated since v0.6.0. This function has been replaced with
-       ``load_sample_data(name="mars_shape")`` and will be removed in
-       v0.9.0.
-
-    This is the ``@mars370d.txt`` dataset used in GMT examples, with data and
-    information from Smith, D. E., and M. T. Zuber (1996), The shape of Mars
-    and the topographic signature of the hemispheric dichotomy. Data columns
-    are "longitude," "latitude", and "radius (meters)."
-
-    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
-    first time you invoke this function. Afterwards, it will load the data from
-    the cache. So you'll need an internet connection the first time around.
+    Load a table of data for the shape of Mars as a pandas.DataFrame.
 
     Returns
     -------
     data : pandas.DataFrame
         The data table with columns "longitude", "latitude", and "radius(m)".
```

Comment:
I feel "radius_m" (with underscore) is a better column name than "radius(m)" (with square brackets).
In case this suggestion is used, please also update line 360 below (GitHub does not allow me to make a code suggestions there):
```python
    data = pd.read_csv(
        fname, sep="\t", header=None, names=["longitude", "latitude", "radius_m"]
```
```suggestion
        The data table with column names "longitude", "latitude", and "radius_m".
```

Reply:
I don't feel strongly either way. We'll see if anyone else weighs in!

Reply:
Ping @GenericMappingTools/pygmt-maintainers for thoughts and comments on this, please 🙂.

Reply:
`radius_m` sounds good to me.